src/Microsoft.Azure.SignalR/HubHost/ServiceLifetimeManager.cs

```diff
@@ -103,6 +116,82 @@ public override async Task SendConnectionAsync(string connectionId, string metho
             }
         }
 
+#if NET7_0_OR_GREATER
```

Comment:
shall we throw NotSupported for WebSocketsHubLifetimeManager?

Reply:
`WebSocketsHubLifetimeManager` inherits `ServiceLifetimeManagerBase` (rather than `ServiceLifetimeManager`)
`ServiceLifetimeManagerBase` inherits `HubLifetimeManager`
In `HubLifetimeManager`, thses methods already throws NotSupported https://source.dot.net/#Microsoft.AspNetCore.SignalR.Core/HubLifetimeManager.cs,147